mail-client/trojita/trojita-0.7.0~git.recipe

```diff
@@ -32,7 +34,10 @@ PROVIDES="
 	"
 REQUIRES="
 	haiku${secondaryArchSuffix}
+#	cmd:gpgconf$secondaryArchSuffix
 	lib:libcrypto$secondaryArchSuffix
+#	lib:libgpgmepp$secondaryArchSuffix
+#	lib:libgpg_error$secondaryArchSuffix
```

Comment:
Could you clarify in the commit message why it is needed to disable gpgme, etc?

Reply:
Because enabling GPGME means QGPGME also needs to be installed and afaik we got no QGPGME.